common/Util.cpp

```diff
@@ -727,6 +731,44 @@ namespace Util
 #endif
     }
 
+    std::string vformatString(const char* format, va_list ap) noexcept
```

Comment:
`noexcept` ? I don't understand.

Reply:
std::string::resize() as used in the implementation is not `noexcept`, i.e. can throw. 
Hence `noexcept` needs to go here indeed. Great catch.